disnake/message.py

```diff
@@ -22,7 +22,7 @@
     overload,
 )
 
-from . import utils
+from . import poll, utils
```

Comment:
This is so insignificant, but the `Poll` class should be imported instead of doing this